app/src/main/res/values/strings.xml

```diff
@@ -516,6 +516,7 @@
     <string name="past_24_hours">Past 24 hours</string>
     <string name="prefer_exoplayer_ffmpeg">Prefer FFmpeg for audio playback</string>
     <string name="prefer_exoplayer_ffmpeg_content">Use FFmpeg to decode audio, even if platform codecs are available.</string>
+    <string name="audio_spdif">SPDIF</string>
```

Comment:
Other name options to better describe this output mode:
ARC
ARC Audio
HDMI Audio

Reply:
Thanks for clarifying the intent of the PR.

To avoid confusion, I'd suggest ARC or maybe even better HDMI ARC.

HDMI Audio is probably too vague as it doesn't differentiate ARC and eARC and would just sound like it means HDMI audio passthrough.

ARC Audio seems a bit redundant as ARC already contains Audio.

Reply:
HDMI ARC seems like the better option, will update  to reflect this.